smallboard/settings.py

```diff
@@ -64,6 +64,7 @@
     "social_django",
     "taggit",
     "rest_framework",
+    "django_celery_results",
```

Comment:
What does this do / are we using the task results anywhere? I think it's also possible to just throw away task results if we don't need them

Reply:
Oh ok, I see that `get_file_user_emails` returns a result, seems reasonable.